README.md

```diff
@@ -1,6 +1,6 @@
 ## dbt-rpc
 
-> :warning: **The `dbt-rpc` plugin will be fully deprecated by the end of 2022.**
+> :warning: **The `dbt-rpc` plugin will be fully deprecated by the second half of of 2023.**
 > 
 > For now, dbt Labs is actively maintaining and using `dbt-rpc` to enable interactive dbt development. Soon, we plan to announce a next-generation dbt Server. After the new Server reaches general release, this plugin will be locked, except for critical maintenance, for a period of six months. After that time, we plan to archive this repository. It will remain available for read-only use.
```

Comment:
Do you want to update this section too? Link to https://github.com/dbt-labs/dbt-server?

This is not quite right, though it's not so far off in the timing:
> After the new Server reaches general release, this plugin will be locked, except for critical maintenance, for a period of six months.

I'd think something to the effect of: We intend to maintain `dbt-rpc` at parity with dbt-core versions up through v1.4. Starting in v1.5, we intend to break ongoing compatibility. We will perform critical maintenance _only_, until the last compatible version of `dbt-core` has reached end of official support (12 months after release; [see policies](https://docs.getdbt.com/docs/dbt-versions/core)).